hax-lib/Cargo.toml

```diff
@@ -12,8 +12,6 @@ description = "Hax-specific helpers for Rust programs"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-hax-lib-macros = { path = "../hax-lib-macros", optional = true }
-
-[features]
-default = ["macros"]
-macros = ["dep:hax-lib-macros"]
+hax-lib-macros = { path = "../hax-lib-macros" }
+num-bigint = { version = "0.4.3", default-features = false, features = [] }
```

Comment:
Using both, `default-features` and `features` seems unnecessary.

Reply:
yeah, sorry, was fighting with CI, fixing that